src/exchange/actions.rs

```diff
@@ -293,3 +293,10 @@ pub struct VaultTransfer {
 pub struct SetReferrer {
     pub code: String,
 }
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
```

Comment:
```suggestion
#[derive(Debug, Clone, Serialize, Deserialize)]
```

Reply:
But every other action has the derive attribute traits in that order. Should I still change it?